sklearn/isotonic.py

```diff
@@ -140,46 +131,31 @@ def isotonic_regression(y, sample_weight=None, y_min=None, y_max=None,
 class IsotonicRegression(RegressorMixin, TransformerMixin, BaseEstimator):
     """Isotonic regression model.
 
-    The isotonic regression optimization problem is defined by::
-
-        min sum w_i (y[i] - y_[i]) ** 2
-
-        subject to y_[i] <= y_[j] whenever X[i] <= X[j]
-        and min(y_) = y_min, max(y_) = y_max
-
-    where:
-        - ``y[i]`` are inputs (real numbers)
-        - ``y_[i]`` are fitted
-        - ``X`` specifies the order.
-          If ``X`` is non-decreasing then ``y_`` is non-decreasing.
-        - ``w[i]`` are optional strictly positive weights (default to 1.0)
-
     Read more in the :ref:`User Guide <isotonic>`.
 
     .. versionadded:: 0.13
 
     Parameters
     ----------
     y_min : float, default=None
-        If not None, set the lowest value of the fit to y_min.
+        Lower bound on the lowest predicted value (the minimum value may
+        still be higher). Defaults to -inf.
```

Comment:
I think it might be confusing that it first says `default=None`, but then `Defaults to -inf`. How about "If not None, lower bound ..." or "If specified, lower bound ..."